modules/overrides/user_config_limits.go

```diff
@@ -0,0 +1,15 @@
+package overrides
+
+type UserConfigurableLimits struct {
+	// TODO how do we ensure Version is always filled in?
+	Version string `json:"version" yaml:"version"`
```

Comment:
if this is an api version we should keep it in the path. that's what we do on other endpoints

Reply:
I've opted to drop the version field for now. Usually when we do a breaking change we just introduce a `/v2/` endpoint. Dealing with versioning even before we know how our api could change might be a bit over eager...